create_cluster.py

```diff
@@ -0,0 +1,99 @@
+import numpy as np
+import dataclasses
+import matplotlib.pyplot as plt
+import seaborn as sns
+import scipy.stats as stats
+import pandas as pd
+import warnings
+import joblib
+import sklearn
+from typing import List
+from sklearn.cluster import KMeans
+from sklearn.datasets import make_blobs
+import json
+from sklearn.utils._openmp_helpers import _openmp_effective_n_threads
+import os
+import datetime
+
+from google.colab import auth
+import gspread
+from google.auth import default  #authenticatiing to google
+from clusters.py import all
+
+def CreateClusterV1(filename: str,
+                    features: List[str],
+                    duplicate_column_name: str,
+                    labels: List[str] = ['R250',	'R500',	'R1000',	'R2000',	'R3000',	'R4000',	'R6000',	'R8000'],
+                    spreadsheet_path: str = spreadsheet_v1_path,
+                    save_path: str = default_cluster_dir,
+                    cluster_labels: dict = {4: 'Low flat',
```

Comment:
We should really not build in this dictionary... a later version of the code that does the cluster assignments might assign each cluster a different name.

Much better to figure out which cluster belongs with each name.  Perhaps using the mean and slope of each cluster as a feature, and then do nearest-neighbor lookup to assign the name.  Make sense?

Reply:
I was thinking of resolving task by task as mentioned on the spreadsheet. Hence, I did not proceed to the step of finding the mean and slope of cluster. I am working on it now. Will commit the changes in a while